src/features/Character/index.js

```diff
@@ -283,8 +283,8 @@ class Character extends Component {
                         onChange={this.hide}
                         label={
                           showPublic
-                            ? `${T.translate('character.shown')}`
```

Comment:
i was confused that the string wasn't translated correctly and i found the reason ^^